Sources/AppStoreConnectCLI/Helpers/String+Helpers.swift

```diff
@@ -10,4 +10,18 @@ extension String {
         return String(String.UnicodeScalarView(filtered))
     }
 
+    func snakeCased() -> String? {
```

Comment:
Any reason we need to bother with snake casing our names? This type of regex matching is prone to breaking when edge cases arise. We can't know how people will name their apps, betaGroups, etc